src/video_core/renderer_vulkan/liverpool_to_vk.cpp

```diff
@@ -281,6 +281,63 @@ vk::BorderColor BorderColor(AmdGpu::BorderColor color) {
     }
 }
 
+const std::vector<vk::Format>& GetAllFormats() {
+    static const std::vector formats{
```

Comment:
```suggestion
    static constexpr std::array formats{
```
Make this a constexpr array and return a std::span<const vk::Format> from this function